### PR TITLE
Apply coding rules

### PR DESCRIPTION
**Pull request by LeeYuRi1**
if문 코딩 룰 오류 수정

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [c0aa18555ff18519f5e9ff8badba4dc49ed6e63b] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [c0aa18555ff18519f5e9ff8badba4dc49ed6e63b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6fb8ab8478a9d9fed3e1905cff02f640cde8899e] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6fb8ab8478a9d9fed3e1905cff02f640cde8899e] - Code Rule Check OK.

File: external/json/cJSON.c [hunk 1]
```diff
@@ -244,31 +234,30 @@ static cJSON_bool parse_number(cJSON * const item, parse_buffer * const input_bu
      * This also takes care of '\0' not necessarily being available for marking the end of the input */
     for (i = 0; (i < (sizeof(number_c_string) - 1)) && can_access_at_index(input_buffer, i); i++)
     {
```
Comment (shadaksharikp) [inline external/json/cJSON.c]: Coding Rule Violation

**Comment by sunghan-chang** [comment]
@LeeYuRi1 This fix is not approprietary. Let's close now and then when you are free, please fix again and reopen this. Thank you for contribution.

**Pull request closed without merging**